firestore/swift/firestore-smoketest/ViewController.swift

```diff
@@ -1229,23 +1236,28 @@ class ViewController: UIViewController {
     }
 }
 
-fileprivate struct City {
+public struct City: Decodable {
 
     let name: String
     let state: String?
     let country: String?
-    let capital: Bool?
+    let isCapital: Bool?
     let population: Int64?
 
-    init?(dictionary: [String: Any]) {
-        guard let name = dictionary["name"] as? String else { return nil }
+    init(name: String, state: String?, country: String?, isCapital: Bool?, population: Int64?) {
```

Comment:
This seems identical to the constructor that the Swift compiler would actually generate for this struct.

Reply:
Removed the extra initializer.